common/tier4_state_rviz_plugin/src/autoware_state_panel.cpp

```diff
@@ -307,20 +309,18 @@ void AutowareStatePanel::onClickAutowareEngage()
 
 void AutowareStatePanel::onClickEmergencyButton()
 {
-  auto request = std::make_shared<tier4_external_api_msgs::srv::SetEmergency::Request>();
-  if (current_emergency_) {
-    request->emergency = false;
-  } else {
-    request->emergency = true;
-  }
+  using SetEmergency = tier4_external_api_msgs::srv::SetEmergency;
+  using ResponseStatus = tier4_external_api_msgs::msg::ResponseStatus;
+
+  auto request = std::make_shared<SetEmergency::Request>();
+  request->emergency = !current_emergency_;
+
   RCLCPP_INFO(raw_node_->get_logger(), request->emergency ? "Set Emergency" : "Clear Emergency");
 
   client_emergency_stop_->async_send_request(
-    request,
-    [this]([[maybe_unused]] rclcpp::Client<tier4_external_api_msgs::srv::SetEmergency>::SharedFuture
-             result) {
-      auto response = result.get();
-      if (response->status.code == tier4_external_api_msgs::msg::ResponseStatus::SUCCESS) {
+    request, [this](rclcpp::Client<SetEmergency>::SharedFuture result) {  // NOLINT
```

Comment:
Why NOLINT?

Reply:
It is the same reason as https://github.com/autowarefoundation/autoware.universe/pull/1589#discussion_r945542757